test/e2e/storage/csi_mock_volume.go

```diff
@@ -793,8 +793,8 @@ func checkPodLogs(cs clientset.Interface, namespace, driverPodName, driverContai
 	if foundAttributes.Len() != 0 {
```

Comment:
let's do this in a follow-up (and keep this PR focused on deflaking), but it is surprising to me that passing expectPodInfo makes this method return early without ensuring node publish and node unpublish are both called

Reply:
yeah that seems like oversight.. I will fix that as a follow up.

Reply:
ping to see if this has been addressed

Reply:
I just opened https://github.com/kubernetes/kubernetes/pull/86891 to fix it.